docs/source/performance.md

```diff
@@ -0,0 +1,329 @@
+<!---
+Copyright 2021 The HuggingFace Team. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# Performance and Scalability: How To Fit a Bigger Model and Train It Faster
```

Comment:
In a title or the preface?

While a lot of parts are pytorch-specific since this is what I know, the hardware is not, and this surely can be expanded to non-pt.

So perhaps to add a preface?

"For now the software sections are mainly pytorch-specific, but the guide might be extended to other frameworks in the future"

Reply:
I added a preface: 

> For now the software sections of this document are mainly Pytorch-specific, but the guide can be extended to other frameworks in the future.

Reply:
Great, thanks!